experiments/ssl4eo/sample_30.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# User-specific parameters
+SAVE_PATH=data/ssl4eo-l-30
+START_INDEX=0
+END_INDEX=10
+
+# Generic parameters
+SCRIPT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+RES=30
+SIZE=264
```

Comment:
We download at 264 so we can center crop or something right?

Reply:
Yep, also to match previous papers